x-pack/plugins/security_solution/public/management/components/endpoint_responder/components/offline_callout.test.tsx

```diff
@@ -60,6 +60,7 @@ describe('Responder offline callout', () => {
       jest.clearAllMocks();
     });
 
+    // TODO TC: include crowdstrike status test
```

Comment:
Are you still planning on adding tests for Crowdstrike in this PR?

Reply:
Thanks for catching this - no I am not planning to add a case for CS here 👍

Reply:
https://github.com/elastic/kibana/pull/180200/commits/5514c114ff92106f8bdcef48913360572cc106b9

Reply:
Ok. I'm assuming that it will be done then in the next PR?  _(Just a reminder that we as a general rule want to include test coverage for the code we commit to `main`)_

Question: is this (the display of the offline callout) supposed to be working as expected? 

Looking at the code for the component, I'm not seeing how it is being displayed and I'm having a bit of trouble testing this out manually, so perhaps you can provide a screen capture of this callout being displayed in the UI for Crowdstrike hosts that are offline?